src/http/interceptors.ts

```diff
@@ -1,8 +1,13 @@
 import { AxiosInstance, AxiosRequestConfig } from 'axios';
```

Comment:
- Check with with FibriCheck if this actually solves their problem
- Check with Fibricheck if the doubleEncoding property solve their problem

Focus should be that it is a breaking version but you can fallback to the previous behaviour.